### ISSUE TITLE
TF/IDF: leave the normalization method as a choice for user?

### ISSUE THREAD

**Issue by jge94** [WheatonCS/Lexos#121]
We imported functions from scikit-learn to get TF/IDF csv matrix & dendrograms, but seems like sklearn uses a different way to calculate TF/IDF than lots of the examples (like Wikipedia) online, for example, idf for sklearn is using natural log, instead of log base 10. But it does have a parameter "norm", which can be set to None OR Manhattan distance OR Euclidean distance. ("norm" is used when a TF/IDF matrix is obtained, and then being normalized into unit vectors row-wise, more details written my comments in getMatrix).
For now I set the default values to be Euclidean distance, with a tooltip next to the button. But since some users might want to get a not-normalized matrix, or prefer Manhattan distance (as we have such a choice in the dendrogram), can we leave this as an option for the user to choose? Maybe three ratio buttons under the "Weighted Counts (TF/IDF)" button, shown only when TF/IDF is chosen? Or do we just want to set it as default?
